### ISSUE TITLE
Improve Accessibility of CreateLink and LinkPlugin

### ISSUE THREAD

**Issue by dharanish-v** [mdx-editor/editor#493]
The Link button in the toolbar and the popup that appears after clicking it both lack an aria-label. Labels need to be added for writing test cases and making the library accessible to screen readers.
